src/transformers/models/data2vec/modeling_data2vec_vision.py

```diff
@@ -0,0 +1,1209 @@
+# coding=utf-8
+# Copyright 2021 Microsoft Research and The HuggingFace Inc. team. All rights reserved.
```

Comment:
```suggestion
# Copyright 2022 Microsoft Research and The HuggingFace Inc. team. All rights reserved.
```

Reply:
Wondering if we should cite both Microsoft (for BEiT) and Facebook (for Data2Vec)

Reply:
Thank just Meta Platforms is good here since they forked BEIT